sentry/src/main/java/io/sentry/Hub.java

```diff
@@ -601,4 +601,57 @@ public void flush(long timeoutMillis) {
     }
     return clone;
   }
+
+  @Override
+  public SentryId captureTransaction(Transaction transaction, Object hint) {
```

Comment:
`hint` is not used anywhere. Should we keep it for the sake of being future-proof or remove and add another method when it's needed?

Reply:
it'd be useful if we have a sort of beforeTransaction callback for filtering.
I dont see any `hint` here though https://develop.sentry.dev/sdk/unified-api/tracing/
cc @HazAT do you know?